pages/sign-in/[[...index]].jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { SignIn } from '@clerk/nextjs'
+export default function SignInPage() {
+  return (
+    <div>
+        <div style={{display:"flex", justifyContent:"center",height:"100vh",alignItems:"center"}}>
```

Comment:
I can change this but since this was only for signin and sign up pages so I didn't create new classname and css in other for this . But if you want I can change  this.

Reply:
It would be great if you can do the same.

Reply:
Ok just doing it now